packages/components/.storybook/preview.ts

```diff
@@ -59,6 +59,10 @@ const preview: Preview = {
           // General
           'Logo',
           'Scrollbar',
+          // Notifications
```

Comment:
This first list is related to the design-system package stories, since both storybooks are composed here. So you don't need to add stories here, since they are from components package and if you can, please change the name `storiesOrder` to `designSystemStoriesOrder`. Besides that, you can move //Notifications prior to // Social and `NotificationItem` before `NotificationsList`